classes/form/CustomerPersister.php

```diff
@@ -167,8 +167,8 @@ private function create(Customer $customer, $clearTextPassword)
             $this->context->updateCustomer($customer);
             $this->context->cart->update();
             $this->sendConfirmationMail($customer);
-            Hook::exec('actionCustomerAccountUpdate', array(
-                'customer' => $customer
+            Hook::exec('actionCustomerAccountAdd', array(
+                'newCustomer' => $customer
```

Comment:
Can you change this back to `customer`, I think it's better to have the same variable name since both hooks are very similar.

Then I'll be happy to merge ;)